galleries/examples/scales/custom_scale.py

```diff
@@ -6,12 +6,21 @@
 ============
 
 Create a custom scale, by implementing the scaling use for latitude data in a
-Mercator Projection.
+Mercator Projection.  A custom scale is useful when
 
-Unless you are making special use of the `.Transform` class, you probably
-don't need to use this verbose method, and instead can use `~.scale.FuncScale`
-and the ``'function'`` option of `~.Axes.set_xscale` and `~.Axes.set_yscale`.
-See the last example in :doc:`/gallery/scales/scales`.
+* You are making special use of the `.Transform` class, such as the special
+  handling of values beyond the threshold in ``MercatorLatitudeTransform``
+  below.
+
+* You want to override the default locators and formatters for the axis
+  (``set_default_locators_and_formatters`` below).
+
+* You want to limit the range of the the axis (``limit_range_for_scale`` below).
+
+If you do not need any of the above, then you probably don't need to use this
+verbose method, and instead can use `~.scale.FuncScale` and the ``'function'``
+option of `~.Axes.set_xscale` and `~.Axes.set_yscale`. See the last example in
```

Comment:
Additional note: I think we should make the FuncScale case more prominent. Not only as "last example in ...". Either as a top level example in parallel to this example, or by making two subsections in this example.

Reply:
I have no objection to this, but also not the bandwidth right now.